### DISCUSSION TITLE
[vweb] Add ssl support

### DISCUSSION THREAD

**Discussion by inxomnyaa** [vlang/v#11702] [Ideas]
Currently it does not seem possible to have vweb running with SSL enabled

It would be great if you could add a way to use/detect installed certificates in vweb

**Comment by JalonSolov** [comment]
These days, it would be better to have auto "Let's Encrypt" support built-in.  That would be support for obtaining a certificate from Let's Encrypt, auto-renewal/replacement on expiration, etc.

Optionally, you should be able to specify a certificate and turn off the Let's Encrypt support.

**Comment by medvednikov** [comment]
I think it's much better to use nginx for this.

It'll be supported, but it's definitely not a priority.
